src/bindings/python/CMakeLists.txt

```diff
@@ -145,12 +136,18 @@ function(ov_check_init_files_alignment)
             RESULT_VARIABLE compare_result
         )
         if(compare_result EQUAL 1)
-            message(FATAL_ERROR "The __init__.py files are misaligned: ${file1} and ${file2}")
+            message(FATAL_ERROR "The runtime __init__.py files are misaligned: ${file1} and ${file2}")
         endif()
     endforeach()
 endfunction()
 
-ov_check_init_files_alignment()
+set(INIT_FILES_RUNTIME
+"${OpenVINOPython_SOURCE_DIR}/src/openvino/__init__.py"
+"${OpenVINOPython_SOURCE_DIR}/src/compatibility/openvino/__init__.py"
+"${OpenVINO_SOURCE_DIR}/tools/ovc/openvino/__init__.py"
+"${OpenVINO_SOURCE_DIR}/tools/benchmark_tool/openvino/__init__.py")
```

Comment:
do we need all these copies at all?
maybe we need to keep only `"${OpenVINOPython_SOURCE_DIR}/src/openvino/__init__.py"` ? All other modules cannot be used without OpenVINO anyway